python/dgl/mock_sparse/sp_matrix.py

```diff
@@ -28,24 +28,24 @@ class SparseMatrix:
     >>> dst = torch.tensor([2, 4, 3])
     >>> val = torch.tensor([1, 1, 1])
     >>> A = SparseMatrix(src, dst, val)
-    >>> A.shape
-    (3,5)
-    >>> A.row
-    tensor([1, 1, 2])
-    >>> A.val
-    tensor([1., 1., 1.])
-    >>> A.nnz
-    3
+    >>> print(A)
+    SparseMatrix(indices=tensor([[1, 1, 2],
+        [2, 4, 3]]),
+    values=tensor([1, 1, 1]),
+    shape=(3, 5), nnz=3)
 
     Case2: Sparse matrix with row indices, col indices and values (vector).
 
     >>> ...
```

Comment:
Remove `...`. We can provide either full example or continuation of the previous case.

Reply:
Done